ote/src/clj/ote/services/register.clj

```diff
@@ -25,50 +34,118 @@
   (if-not (valid-registration? form-data)
     ;; Check errors that should have been checked on the form
     {:success? false}
-    (with-transaction db
```

Comment:
transaction might still be needed? or is this called from inside an outer transaction?